metric-schema-java/src/integrationInput/java/com/palantir/test/VisibilityMetrics.java

```diff
@@ -39,7 +39,7 @@ Counter test() {
 
     /** Tagged gauge metric. */
     @CheckReturnValue
-    ComplexBuilderFooStage complex() {
+    ComplexBuilderBarStage complex() {
```

Comment:
I don't think we should reorder builder stages, this would require a great deal of manual effort for large internal projects that heavily utilize metric-schema.

Originally I think we agreed to preserve tag definition order from the input. I suspect this is the result of a non-linked set.